src/main/java/ch/ivyteam/ivy/maven/MavenDependencyMojo.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2018 AXON Ivy AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ch.ivyteam.ivy.maven;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+import ch.ivyteam.ivy.maven.engine.MavenProjectBuilderProxy;
+
+/**
+ * Copy maven dependencies to a specific folder.
+ * 
+ * @since 9.2.0
+ */
+@Mojo(name=MavenDependencyMojo.GOAL, requiresDependencyResolution=ResolutionScope.COMPILE)
+public class MavenDependencyMojo extends AbstractProjectCompileMojo
+{
+  public static final String GOAL = "maven-dependency";
+  
+  /** 
+   * Set to <code>true</code> to bypass the copy of <b>maven dependencies</b>.
+   */
+  @Parameter(property="ivy.mvn.dep.skip", defaultValue="false")
+  boolean skipMvnDependency;
+  
+  int copied = 0;

```

Comment:
strange to have this simple counter as mojo field: I'd rather return the count from 'copyDepdendency' and stick to a Collection<Input> ... would be easier to read